### PR TITLE
original GMM for motion detection

### PR DESCRIPTION
**Pull request by wany1601**
Original GMM for motion detection

### CODE REVIEW

File: machine-learning/_posts/2017-03-30-GMM.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+tags:   motion-detection segmentation essentials
+author: Yi Wang
+pdf:    http://profs.sci.univr.it/~cristanm/teaching/sar_files/lezione4/vsam-pami-tracking.pdf
+cite:
+  authors: "Stauffer, Chris and Grimson, W. Eric L."
+  title:   "Learning patterns of activity using real-time tracking"
+  venue:   "PAMI, 2000"
+---
+
+Gaussian mixture model (GMM) is one of the most popular motion detection models. As a parametric models, GMM generates multiple Gaussian models for each pixel in the video. When a new frame arrives, GMM compares each pixel in the frame with every Gaussian model in the mixture until a matching Gaussian model is found. The matched Gaussian will be updated, and the pixel will be considered to be a background. If no Gaussian model mateched, the least probable distribution will be replaced with a distribution with its mean equales to the current pixel value, and the pixel will be considered to be a foreground.
```
Comment (lemairecarl) [inline machine-learning/_posts/2017-03-30-GMM.md]: Some corrections to make:

* As a parametric models -> As a parametric model
* will be considered to be a background -> to be from the background
* If no Gaussian model mateched -> matched
* distribution with its mean equales to -> distribution with a mean equal to
* considered to be a foreground -> considered to be from the foreground

Comment (jodoin) [inline machine-learning/_posts/2017-03-30-GMM.md]: Plaese add a GMM tag in the 2017-03-30-GMM.md file as well as in the how-to file.

Comment (jodoin) [inline machine-learning/_posts/2017-03-30-GMM.md]: Also, I dont think this paper should be put in the "machine learning" category.  I would prefer to put it into a "video-analysis" category.

File: video-analysis/_posts/2017-03-30-GMM.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+tags:   motion-detection segmentation essentials GMM
+author: Yi Wang
+pdf:    http://profs.sci.univr.it/~cristanm/teaching/sar_files/lezione4/vsam-pami-tracking.pdf
+cite:
+  authors: "Stauffer, Chris and Grimson, W. Eric L."
+  title:   "Learning patterns of activity using real-time tracking"
+  venue:   "PAMI, 2000"
+---
+
+Gaussian mixture model (GMM) is one of the most popular motion detection models. As a parametric model, GMM generates multiple Gaussian models for each pixel in the video. When a new frame arrives, GMM compares each pixel in the frame with every Gaussian model in the mixture until a matching Gaussian model is found. The matched Gaussian will be updated, and the pixel will be considered to be a background pixel. If no Gaussian model mateched, the least probable distribution will be replaced with a new distribution with its mean equales to the current pixel value, and the pixel will be considered to be a foreground pixel.
```
Comment (lemairecarl) [inline video-analysis/_posts/2017-03-30-GMM.md]: There are two typos that you did not correct: "mateched" and "equales" (should be "matched" and "equal")

**Pull request merged**